CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.3.0
```

Comment:
Would you mind just bumping the version to `1.2.1`? This feels like a bug fix to me.